tests/bdd/features/volume/topology/node-topology.feature

```diff
@@ -0,0 +1,73 @@
+    which signifies the volume node topology inclusion labels as {"A1": "B1", "C E" : "D1"}
+
+    nodeHasTopologyKey: |
+      alpha
+      beta    
+   which signifies the volume node topology inclusion labels as {"alpha": "", "beta" : ""} 
+
+  Background:
+    Given a control plane, four Io-Engine instances, twelve pools
+
+
+# The labels to be applied to the nodes.
+#################################################################################################
+#   Description              ||  Node Name             ||    Pool Name   ||     Node Label     ||
+#==============================================================================================
+#     "io-engine-1" has      ||   node1 (io-engine-1)  ||    node1pool1  ||                    ||
+#  label "alpha1=beta1"      ||   node1 (io-engine-1)  ||    node1pool2  ||     alpha1=beta1   ||
+#                            ||   node1 (io-engine-1)  ||    node1pool3  ||                    ||
+#==============================================================================================
+#     "io-engine-2" has      ||   node2 (io-engine-2)  ||    node2pool1  ||                    ||
+#  label "alpha2=beta2"      ||   node2 (io-engine-2)  ||    node2pool2  ||     alpha2=beta2   ||
+#                            ||   node2 (io-engine-2)  ||    node2pool3  ||                    ||
+#==============================================================================================
+#     "io-engine-3" has      ||   node3 (io-engine-3)  ||    node3pool1  ||                    ||
+#  label "alpha3=beta3"      ||   node3 (io-engine-3)  ||    node3pool2  ||     alpha3=beta3   ||
+#                            ||   node3 (io-engine-3)  ||    node3pool3  ||                    ||
+#==============================================================================================
+#     "io-engine-4" has      ||   node4 (io-engine-4)  ||    node4pool1  ||     alpha1=beta1   ||
+#  label "alpha1=beta1,      ||   node4 (io-engine-4)  ||    node4pool2  ||     alpha2=beta2   ||
+# alpha2=beta2,alpha3=beta3" ||   node4 (io-engine-4)  ||    node4pool3  ||     alpha3=beta3   ||
+#================================================================================================
+
+
+  Scenario Outline: Suitable nodes which contain volume topology labels
+    Given a request for a <replica> replica volume with nodeAffinityTopologyLabel as <node_affinity_topology_label> and node topology inclusion as <volume_node_topology_inclusion_label>
+    When the desired number of replica of volume i.e. <replica> here; is <expression> number of the nodes having the label <volume_node_topology_inclusion_label>
+    Then the <replica> volume creation should <result> and <provisioned> provisioned on pools with its corresponding node labels <node_label>
+    Examples:
+      | node_affinity_topology_label | volume_node_topology_inclusion_label  | replica |     expression        | result    | provisioned |  node_label    |
```

Comment:
The first columns here seem redundant?

Reply:
Yes, but it was added that we can add any test case where in `node_affinity_topology_label` can be false.